lib/service.js

```diff
@@ -21,17 +21,31 @@ export default class WdioImageComparisonService extends BaseClass {
      * @param {Array.<Object>} capabilities list of capabilities details
      */
     before(capabilities) {
-        const instanceData = getInstanceData(capabilities)
+        if(typeof capabilities['browserName'] !== 'undefined') {
+            console.log('Adding to global browser')
+            this.addCommandsToBrowser(capabilities, browser)
+        } else {
+            const browserNames = Object.keys(capabilities)
+            console.log('Adding to multibrowser: ', browserNames)
+            for(const browserName of browserNames) {
+                this.addCommandsToBrowser(capabilities[browserName], global[browserName])
+            }
+        }
+    }
+
+    addCommandsToBrowser(capabilities, currentBrowser) {
```

Comment:
I need to dive into this more, but with the previous and current solution `browser.saveScreen('examplePage');` breaks with this error

```log
TypeError: browser.saveScreen is not a function
```

So before I merge this I need to have a solution for calling the commands on the multiRemote instance instead of a single multiRemote instance

Reply:
Yes, since we don't add the commands to the overall `browser` in the Multi Remote case, it will not function to do this for the Multi Remote config. I can add commands to `browser` in that case that would just call it on each capability if you like that idea? 

From their doc:
> All commands your tests call via the browser variable are executed in parallel with each instance.
https://webdriver.io/docs/multiremote/

Reply:
I don't think it will work when you add it to the overall `browser`. It gave the same error as without your changes

Reply:
I'm working on this solution now. I'll let you know. I'll commit the test changes for now at least.

Reply:
I've pushed the solution for this. If the code is too repetitive, let me know and I can look into calling it for each function name.

Reply:
Note I also added to the types file to reflect returning either the regular output or the Object with the output per browser.